v21.1/create-changefeed.md

```diff
@@ -148,6 +150,23 @@ CockroachDB Type | Avro Type | Avro Logical Type
 [`INET`](inet.html) | [`STRING`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
 [`JSONB`](jsonb.html) | [`STRING`](https://avro.apache.org/docs/1.8.1/spec.html#schema_primitive) |
 
+#### Topic Naming
+
+By default, a Kafka topic has the same name as the table that a changefeed was created on. If a changefeed was created on multiple tables, the changefeed will write to multiple topics corresponding to those table names. You can specify a [topic prefix](#query-parameters) or use the [`full_table_name` option](#options) to modify this.
+
+You can either manually create a topic in your Kafka cluster before starting the changefeed, or the topic will be automatically created when the changefeed connects to your Kafka cluster. 
+
+{{site.data.alerts.callout_info}}
+You must have the Kafka cluster setting `auto.create.topics.enable` set to `True` for automatic topic creation.
```

Comment:
When using automatic topic creation, the topic will not be created until the changefeed sends its first message. If you create the consumer before that, you will also need the `allow.auto.create.topics` consumer setting.

Reply:
Done.